biomage/experiment/pull.py

```diff
@@ -99,10 +120,17 @@ def pull(experiment_id, origin):
     """
 
     Summary.set_command(cmd=PULL, origin=origin, experiment_id=experiment_id)
+
     bucket = f"biomage-source-{origin}"
-    file = f"{experiment_id}/r.rds"
+    file = f"{experiment_id}/{RDS_FILE}"
+    dst_file = file + ".gz"
+    download_if_modified(bucket=bucket, key=file, filepath=dst_file)
```

Comment:
the name of the bucket here and in line 133 are different, even though we put in the function 
 calls of `download_if_modified` `bucket=bucket` in both places. What this means that if we swapped the two lines of code, technically the code will run correctly, but the results will be wrong in a subtle way. I can think of these ways in mitigating this:
 - directly put the bucket name in each of the calls to `download_if_modified` and not have a `bucket` variable defined
 - extract out the two different bucket names as different variables and put them at the top of the file similarly to what is done for `PLOTS_TABLES_FILE`, `CELLSETS_FILE`, etc. for example. Then in `download_if_modified` refer to the relevant bucket name.
 - less effective, but you can add a comment above each call to `download_if_modified` 
What do you think?